libs/blocks/src/lib/features/card-feature/index.tsx

```diff
@@ -35,7 +37,7 @@ const Card = ({
   return (
     <Section
       className={clsx(
-        'py-general-4xl',
+        hasPadding ? 'py-general-4xl' : 'py-general-xl',
```

Comment:
Can this be overridden using `className`? Instead of adding a new props for it.

Reply:
It is taking `py-general-4xl` as default padding, may be we need to give `!important`